packages/dart/lib/src/base/parse_constants.dart

```diff
@@ -1,4 +1,4 @@
-part of flutter_parse_sdk;
+part of '../../parse_server_sdk.dart';
```

Comment:
Why this change?

Reply:
It is recommended to use this method in new versions of [Lint](https://pub.dev/packages/flutter_lints).
If we don't use it, we will get a warning and we will have problems in ci !